tests.py

```diff
@@ -261,6 +261,14 @@ class PsTestCase(BaseTestCase, PathAndFileTests):
     ext = "ps"
 
 
+class EmlTestCase(BaseTestCase, PathAndFileTests):
+    ext = "eml"
+
+
+class MboxTestCase(BaseTestCase, PathAndFileTests):
+    ext = "mbox"
+
+
```

Comment:
No `.msg`?

Reply:
I don't have a way to create a test file just yet. I should soon, and I will add it and merge.